examples/tree_2d_dgsem/elixir_advection_restart.jl

```diff
@@ -32,8 +32,7 @@ integrator = init(ode, alg,
                   save_everystep=false, callback=callbacks)
 
 # Get the last time index and work with that.
-integrator.iter = load_timestep(restart_filename)
-integrator.stats.naccept = integrator.iter
+load_timestep!(integrator, restart_filename)
```

Comment:
Is this the only restart file where we test this? Please check that we do not use the old way in any of the examples anymore.

Reply:
I changed it in all the other examples and also in documentation